src/lib/readmeAPIFetch.ts

```diff
@@ -322,6 +322,7 @@ export async function readmeAPIv2Fetch<T extends Hook.Context = Hook.Context>(
     })
     .catch(e => {
       this.debug(`error making fetch request: ${e}`);
+      this.debug(e.stack);
```

Comment:
Added this in because it was annoying only setting `fetch failed` errors with no additional context.